vm/src/bytesinner.rs

```diff
@@ -24,6 +23,7 @@ use crate::{
     anystr::{self, AnyStr, AnyStrContainer, AnyStrWrapper},
     obj::objtuple::PyTuple,
 };
+use crate::{byteslike::try_bytes_like, obj::objbytes::PyBytesRef};
```

Comment:
`PyBytesRef` can be grouped with `PyBytes` in line 7

Reply:
It is rust-analyzer's auto import, forget to fix it.

Reply:
wow, i didn't know it has auto import feature